elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ElytronOidcDescriptionConstants.java

```diff
@@ -55,6 +57,16 @@ interface ElytronOidcDescriptionConstants {
     String REGISTER_NODE_AT_STARTUP = "register-node-at-startup";
     String REGISTER_NODE_PERIOD = "register-node-period";
     String REPLACEMENT = "replacement";
+    String REQUEST = "request";
+    String REQUEST_OBJECT_CONTENT_ENCRYPTION_ALGORITHM = "request-object-content-encryption-algorithm";
```

Comment:
Does this need to be renamed to align with the changes made in the Elytron PR?

Reply:
Ah, I think I see now, these changes are part of the documentation commit. I think it would be good to squash the documentation and implementation commits together.

Reply:
Just the subsystem bump can be a separate commit. Thanks!

Reply:
I have rebased it to fix that. 
The name changes should be reflected in the RFE code too now.